sdk/angelscript/projects/cmake/CMakeLists.txt

```diff
@@ -178,37 +178,38 @@ install(TARGETS ${ANGELSCRIPT_LIBRARY_NAME} EXPORT AngelscriptTargets
 
 install(FILES
 	${CMAKE_CURRENT_SOURCE_DIR}/../../include/angelscript.h
-	DESTINATION include
+	DESTINATION include/${ANGELSCRIPT_LIBRARY_NAME}
 	COMPONENT Devel
 )
 
+
 include(CMakePackageConfigHelpers)
 write_basic_package_version_file(
-	"${CMAKE_CURRENT_BINARY_DIR}/Angelscript/AngelscriptConfigVersion.cmake"
+	"${CMAKE_CURRENT_BINARY_DIR}/Angelscript/${ANGELSCRIPT_LIBRARY_NAME}ConfigVersion.cmake"
```

Comment:
The useless change, not fixing anything or not introducing any new functionality.

Reply:
Well you think so? I just had problems because of hardcoded names when the project itself introduces a variable to refer to library name which you can change but then all files generated are wrong because they're well hardcoded. I'm starting to loose the picture of what you consider worthy change, or not even sure I've ever grasped it. We're talking about additional CMakeFiles changes compared to original files because Hunter won't work properly without them - this is already as bad as it could be because we have to keep separate fork. This project has not been updated and the version offered by Hunter is some 5 year old unofficial fork that had nothing to do with orignal source code, yet we're ping-ponging ideal-world-CMakeFiles-patch ideas here. 

I'm not sure I have time for guessing what you'd consider a small and worthy change. Not sure I have time for this really for guessing your arbitrary views on this. TL;DR I think this is making the whole thing . consistent vs. using var in one place and hardcoded names elsewhere.